core/services/ocr2/plugins/ccip/internal/ccipdata/v1_0_0/offramp.go

```diff
@@ -92,6 +93,57 @@ func (d ExecOnchainConfig) PermissionLessExecutionThresholdDuration() time.Durat
 	return time.Duration(d.PermissionLessExecutionThresholdSeconds) * time.Second
 }
 
+// An ExecOffchainConfig is the configuration for nodes executing committed CCIP messages (v1.0–v1.2).
+// It comes from the OffchainConfig field of the corresponding OCR2 plugin configuration.
+// NOTE: do not change the JSON format of this struct without consulting with the RDD people first.
+type ExecOffchainConfig struct {
+	// SourceFinalityDepth indicates how many confirmations a transaction should get on the source chain event before we consider it finalized.
+	SourceFinalityDepth uint32
+	// See [ccipdata.ExecOffchainConfig.DestOptimisticConfirmations]
+	DestOptimisticConfirmations uint32
+	// SourceFinalityDepth indicates how many confirmations a transaction should get on the destination chain event before we consider it finalized.
```

Comment:
Typo: `DestFinalityDepth`
```suggestion
	// DestFinalityDepth indicates how many confirmations a transaction should get on the destination chain event before we consider it finalized.
```